eth/rollup.go

```diff
@@ -60,7 +60,7 @@ type RollupEventInitialize struct {
 
 // RollupVariables returns the RollupVariables from the initialize event
 func (ei *RollupEventInitialize) RollupVariables() *common.RollupVariables {
-	var buckets [common.RollupConstNumBuckets]common.BucketParams
+	buckets := make([]common.BucketParams, 1)
```

Comment:
In the initialization of `InstantWithdrawManager.sol` (where the buckets are defined), the buckets are not touched.  This means that on initalization, there are 0 buckets initially, so buckets should be an empty slice. Example
```go
buckets := []common.BucketsParams{}
```

https://github.com/hermeznetwork/contracts/blob/d439b23a81e64f6cd65869022d06d7ef85729632/contracts/hermez/Hermez.sol#L223
https://github.com/hermeznetwork/contracts/blob/d439b23a81e64f6cd65869022d06d7ef85729632/contracts/hermez/lib/InstantWithdrawManager.sol#L58

Reply:
Fixed